README.md

```diff
@@ -3,3 +3,4 @@
 This repository contains a Docker image with the [R `spelling` package](https://cran.r-project.org/web/packages/spelling/index.html) installed.
 The role of this repository is to facilitate spell checking actions across `AlexsLemonade` repositories.
 
+speeling
```

Comment:
I think the important question here is whether we want to just have this here, or integrate it more sneakily. Like, 
> The role of this repository is to facilitate speell checking actions across `AlexsLemonade` repositories.

Either way, it must stay in some capacity.